test/options.test.js

```diff
@@ -2,14 +2,17 @@ import chai, { expect } from 'chai'
 import promised from 'chai-as-promised'
 chai.use(promised)
 import path from 'path'
+require('json5/lib/require')
```

Comment:
This just adds into require.extensions the `json5` format as a hook. 
At the end it's just `fs.readFileSync` the files with the correct format, the same as node does with normal JSON or other JS files.